seesaw/configs.py

```diff
@@ -77,4 +77,14 @@
     "coco": "coco",
```

Comment:
I think we can remove these mappings. It used to be that we had a mapping for names used in the URL to actual dataset names because these could be different. as we add more datasets and indices it is easier if we stick to a convention that folder name for dataset = dataset name, and folder name for index = index name. this will also keep things consistent between url and file system, which is helpful to us (we need to name the indices in ways that dont need to be especially encoded in url though). can you make this change (remove these maps and just use the url directly